content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -22,6 +22,15 @@ Kubernetes, on Linux:
 - [CRI-O](#cri-o)
 - [Docker](#docker)
 
+Dockershim has been [deprecated](https://kubernetes.io/blog/2020/12/02/dockershim-faq/)
+as of [version 1.20.0](https://github.com/kubernetes/kubernetes/pull/94624)
+and is likely to be dropped of
+by 1.24.0, therefore to use Docker you need to be running an older version of
+Kubernetes. You can check out this [documentation](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/)
+to understand how this deprecation might affect you. For migrating from dockershim you can follow
+[this](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/_print/).
+A robust guide on migrating from Dockershim to other CRIs will be released soon.
+
```

Comment:
```suggestion
Dockershim has been [deprecated](https://kubernetes.io/blog/2020/12/02/dockershim-faq/)
as of [version 1.20.0](https://github.com/kubernetes/kubernetes/pull/94624)
and is likely to be dropped of by 1.24.0, therefore to use Docker you need to be running an older version of
Kubernetes. 
To know how this deprecation affects you and how to migrate from dockershim you can follow [this document](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/_print/).
```

Reply:
Also its better if we add this in `notes` section

Reply:
@adisky, should I create another note section for this or add it in the existing one?